catalyst/dl/experiment/experiment.py

```diff
@@ -143,6 +143,30 @@ def distributed_params(self) -> Dict:
         """Dict with the parameters for distributed and FP16 method."""
         return self._distributed_params
 
+    @property
+    def hparams(self) -> OrderedDict:
+        """Returns hyper parameters"""
```

Comment:
btw, could you please also add an example output here?
like https://github.com/catalyst-team/catalyst/blob/24f147f52a25612a88b1e94df7a89ee31619caa4/catalyst/core/experiment.py#L77#L78